### ISSUE TITLE
Multiwallet

### ISSUE THREAD

**Issue by juscamarena** [ruimarinho/bitcoin-core#83]
Multiwallet support does not support using commands through .command for new rpcs.

For example this command below will fail even with a wallet set.

Checking debug logs shows that in cases with manual commands the wallet path is not added to the url.

client.command('walletprocesspsbt', rawPbstTxn)

**Comment by pedrobranco** [comment]
By the actual implementation, to use that wallet path it is required to be defined in `methods.js` file, and enable that feature.

Don't know if it is worth to have a `walletCommand(wallet, ...args)` to allow those calls.

**Comment by juscamarena** [comment]
Yeah, I manually added them to the methods.js as a workaround in a separate branch I kept,  I think we should have an walletCommand(wallet, ...args) to allow those calls otherwise this lib needs to be constantly updated ever release to use any new multiwallet commands.
